code/modules/crafting/recipes.dm

```diff
@@ -84,10 +84,12 @@
 
 /datum/table_recipe/flamethrower
 	name = "Flamethrower"
-	result = /obj/item/weapon/flamethrower
+	result = /obj/item/weapon/flamethrower/full
```

Comment:
Since the /full child of flamethrower is always created with a generic welder, this'll mean when you deconstruct a tablecrafted flamethrower made with an experimental or industrial welder you'll lose that.